web/web.go

```diff
@@ -263,7 +265,20 @@ func (h *Handler) statush(w http.ResponseWriter, r *http.Request) {
 	h.status.mu.RLock()
 	defer h.status.mu.RUnlock()
 
-	h.executeTemplate(w, "status", h.status)
+	h.executeTemplate(w, "status", struct {
+		Status *PrometheusStatus
+		Info   map[string]string
+	}{
+		Status: h.status,
+		Info:   version.Map,
+	})
+}
+
+func (h *Handler) version(w http.ResponseWriter, r *http.Request) {
```

Comment:
Ok, seems fine to me for now. We can always add/move it to the v1 API later if needed, because we're not giving any guarantees for anything outside of it :)